cms/static/cms/js/modules/cms.changeform.js

```
@@ -13,22 +14,32 @@ $(document).ready(function () {
 
     // hide rows when hidden input fields are added
     $('input[type="hidden"]').each(function () {
-        $(this).parent('.form-row').hide()
+        $(this).parent('.form-row').hide();
     });
 
     // determine if slug is empty
-    if(slug.val() === '') prefill = true;
+    if(slug.val() === '') {
+        prefill = true;
+    }
 
     // always bind the title > slug generation and do the validation inside for better ux
     title.bind('keyup', function() {
         var value = title.val();
         // international language handling
-        if(window.UNIHANDECODER) value = UNIHANDECODER.decode(value);
+        if(window.UNIHANDECODER) {
+            value = UNIHANDECODER.decode(value);
+        }
         // if slug is empty, prefill again
-        if(prefill === false && slug.val() === '') prefill = true;
+        if(prefill === false && slug.val() === '') {
+            prefill = true;
+        }
         // urlify
+        // jshint newcap: false
         var urlified = URLify(value, 64);
-        if(prefill) slug.val(urlified);
+        // jshint newcap: true
+        if(prefill) {
```

that should also throw an error?